spec/requests/admin/users_requests_spec.rb

```diff
@@ -128,7 +128,7 @@
       end
 
       it "preloads organizations" do
-        post admin_users_path, params: { user: { organization_id: 1 } }
+        post admin_users_path, params: { user: { organization_id: 1, email: 'hello@gm.com' }}
```

Comment:
I think this is not needed -- please undo this change (or share what it is needed for).